setup.py

```diff
@@ -35,4 +24,10 @@
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
     ],
+    python_requires=">=3.7",
+    install_requires=["conda-package-streaming >= 0.5.0", "tqdm"],
+    extras_require={
+        "docs": ["furo", "sphinx", "myst-parser", "mdit-py-plugins>=0.3.0"],
+        "test": ["mock", "pytest", "pytest-cov", "pytest-mock"],
```

Comment:
@travishathaway these names are common e.g https://github.com/sphinx-doc/sphinx/blob/master/pyproject.toml#L78

One for running sphinx, the other for running pytest.